p2p/src/sync/peer_v2/transaction_manager.rs

```diff
@@ -49,13 +50,18 @@ use crate::{
 
 use super::requested_transactions::RequestedTransactions;
 
+// TODO: add smaller interval for outbound connections
+pub const TX_RELAY_DELAY_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
Why should there be a minimum amount and not start from zero?

Reply:
it's mean value not minimum